src/main/java/com/rackspace/salus/telemetry/etcd/services/ZoneStorage.java

```diff
@@ -220,16 +248,16 @@ private int constrainNewCount(int newCount,
   }
 
   /**
-   * Retrieves the latest written revision version of the expected events key.
+   * Retrieves the latest written revision version of the events key.
    *
    * See {@link com.rackspace.salus.telemetry.etcd.types.Keys} for a description of how the tracking
    * key is used.
    *
    * @return A completable future containing the revision version, or 0 if the key is not found.
    */
-  private CompletableFuture<Long> determineWatchSequenceOfExpectedZones() {
+  private CompletableFuture<Long> getRevisionVersionOfKey(String key) {
```

Comment:
...so this should probably just be

```suggestion
  private CompletableFuture<Long> getRevisionOfKey(String key) {
```